src/components/index.js

```diff
@@ -1,5 +1,9 @@
+import React from 'react'
+import 'antd/dist/antd.css'
+import TodoList from './TodoList'
+
 const App = () => {
-  return <h2>HElloo</h2>
+  return <TodoList />
```

Comment:
You don`t need to use it now in App, because if I merge your pull-request that's create merge-conflict in another pull-request.